content/docs/concepts/webhook.md

```diff
@@ -1,32 +1,34 @@
 ---
-title: Webhook
-description: 'cert-manager core concepts: Webhook'
+title: All About the cert-manager Webhook
+description: |
+    Learn about the webhhook component of cert-manager, which validates, converts and sets default values for the cert-manager custom resources
 ---
 
-cert-manager makes use of extending the Kubernetes API server using a Webhook
-server to provide [dynamic admission
-control](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
-over cert-manager resources. This means that cert-manager benefits from most of
-the same behavior that core Kubernetes resource have. The webhook has three
-main functions:
+cert-manager extends the Kubernetes API using Custom Resource Definitions
+and it installs a webhook which has three main functions:
```

Comment:
I'm not a fan of `and it` here.

One approach would be:
```suggestion
cert-manager extends the Kubernetes API using Custom Resource Definitions.
It installs a webhook which has three main functions:
```

Note that `functions` can have two meanings, one `a programming function` and the other roughly `purpose/task`. I think you're using it in the latter meaning, but since we're programmers first, I think people are more likely to parse it in the former. That said, I can't think of a word I like better here for the latter meaning.

Reply:
Duties? I'll leave it as "functions" for now.